src/sb.cpp

```diff
@@ -57,4 +58,26 @@ int Sb::CastOffEncoding(FunctorParams *functorParams)
     return FUNCTOR_SIBLINGS;
 }
 
+int Sb::CastOffSystems(FunctorParams *functorParams)
```

Comment:
Functor methods should always return a value (the PR currently fails because of this missing)